code/game/verbs/who.dm

```diff
@@ -58,7 +58,7 @@
 
 	msg += "<b>Total Players: [length(Lines)]</b>"
 	to_chat(src, msg)
-
```

Comment:
Delete this old code

Reply:
That's not old code. That belongs to regular Who verb. Not staffwho.

My changes focused on staffwho.

Reply:
the code right here says staffwho. line 62 onwards